app/assets/javascripts/app/components/comment_form/templates/desktop.html.slim

```diff
@@ -1,10 +1,15 @@
 .CardList layout="column"
   md-card.Comment id="newComment"
-    textarea#js-comment-content.Editor placeholder="State your arguments" ng-model="ctrl.comment.body"
+    textarea.Editor.Editor--card msd-elastic="\n" placeholder="State your arguments" ng-model="ctrl.comment.body"
 
-    md-toolbar md-theme="white" on-scroll="" on-scroll-anchor="#js-comment-content" scrolling-class="md-whiteframe-z1" layout="row" layout-align="end center"
-      council-raised-button.Button.Button--secondary disabled="ctrl.disabled" click="ctrl.create()" ng-hide="ctrl.comment.id"
-        | Add Comment
+    .ActionBar
```

Comment:
This component purpose is to wrap all the action buttons on the comment card, like edit, comment, update and so on.